### ISSUE TITLE
Binary Release in Release Tab

### ISSUE THREAD

**Issue by AdnanHussainTurki** [databacker/mysql-backup#334]
Hi Team,

Can we please put up the binary releases in the GitHub's releases tab too? It would be essential for lazy guys like me to just download and use as executable :P

Thanks :D

**Comment by deitch** [comment]
I thought there were binaries pushed as release assets, and I was right. It is in the flow. But it is not working. I am going to cut a temporary release just to see.

**Comment by deitch** [comment]
Ah, it does do it, but it is not finding the created binary files, [logs](https://github.com/databacker/mysql-backup/actions/runs/10162361449/job/28102868526#step:5:10):

```
  with:
    files: dist/*
  
    token: ***
  env:
    IMAGE_NAME: databack/mysql-backup
    GOROOT: /opt/hostedtoolcache/go/1.21.12/x64
🤔 Pattern 'dist/*' does not match any files.
👩‍🏭 Creating new GitHub release for tag v1.0.0-rc5...
🤔 dist/* not include valid file.
```

**Comment by deitch** [comment]
And fixed! Check out the releases v1.0.0-rc5, which has all of the binaries.
